src/bundle/upload.ts

```diff
@@ -395,6 +395,41 @@ async function setVersionInChannel(
   }
 }
 
+async function getBundleSignature(options: Options, config: CapacitorConfig, bundle: Buffer): Promise<string | null> {
```

Comment:
There is a problem here, signature is mandatory. This will break all CI/CD of every users.
Make it optional

Reply:
It is optional, `log.info('Private key not found, skipping signing')`